### PR TITLE
Hardware API: updates tables with consistent capitalization

### PR DESCRIPTION
**Pull request by HipsterBrown**
Following the recent docs update, the table headers should be capitalized along with the first word in the "Description" and "Notes" columns.

### CODE REVIEW

File: API/Hardware_API.md [hunk 1]
```diff
@@ -195,13 +195,13 @@ pin.once(eventName, callback);
 
 #### Available Events
 
-eventName | description           | notes
+EventName | Description           | Notes
```
Comment (Frijol) [inline API/Hardware_API.md]: Why no space?

Comment (Frijol) [inline API/Hardware_API.md]: (in EventName)

File: API/Hardware_API.md [hunk 2]
```diff
@@ -221,9 +221,9 @@ Set the signal frequency in hertz. 1 hertz equals 1 cycle of signal per second.
 tessel.pwmFrequency(number);
 ```
 
-option | type   | description                         | required
+Option | Type   | Description                         | Required
 -------|--------|-------------------------------------|---------
-number | Number | minimum value 1, maximum value 5000 | yes
+number | Number | Minimum value 1, maximum value 5000 | yes
```
Comment (Frijol) [inline API/Hardware_API.md]: 'Yes' should be capitalized

File: API/Hardware_API.md [hunk 3]
```diff
@@ -236,9 +236,9 @@ Set the percentage amount of time the pin is active each cycle of signal.
 pin.pwmDutyCycle(number);
 ```
 
-option | type   | description                                | required
+Option | Type   | Description                                | Required
 -------|--------|--------------------------------------------|---------
-number | Number | minimum value 0, maximum value 1, e.g. 0.6 | yes
+number | Number | Minimum value 0, maximum value 1, e.g. 0.6 | yes
```
Comment (Frijol) [inline API/Hardware_API.md]: 'Yes'

File: API/Hardware_API.md [hunk 4]
```diff
@@ -406,9 +406,9 @@ Allow power to one or both Tessel ports.
 tessel.open(portName);
 ```
 
-option   | type   | description       | required
+Option   | Type   | Description       | Required
 ---------|--------|-------------------|---------
-portName | String | either 'A' or 'B' | no
+portName | String | Either 'A' or 'B' | no
```
Comment (Frijol) [inline API/Hardware_API.md]: 'No'

File: API/Hardware_API.md [hunk 5]
```diff
@@ -420,9 +420,9 @@ Disallow power to one or both Tessel ports.
 tessel.close(portName);
 ```
 
-option   | type   | description       | required
+Option   | Type   | Description       | Required
 ---------|--------|-------------------|---------
-portName | String | either 'A' or 'B' | no
+portName | String | Either 'A' or 'B' | no
```
Comment (Frijol) [inline API/Hardware_API.md]: 'No'

**Comment by HipsterBrown** [comment]
Updated @Frijol. Thanks for the review!

**Pull request merged**